src/main/resources/db/migration/0250/V282__DocumentProducerTables.sql

```diff
@@ -24,6 +29,12 @@ CREATE TABLE docprod.variables (
     stable_id TEXT NOT NULL,
     name TEXT NOT NULL,
     description TEXT,
+    deliverable_id BIGINT REFERENCES accelerator.deliverables ON DELETE SET NULL,
+    deliverable_question TEXT,
+    dependency_variable_stable_id TEXT,
```

Comment:
We'll need to keep track of the order of the questions.

Reply:
I'm planning on adding that in another PR since this one has already ballooned quite a bit.